README.md

```diff
@@ -3,18 +3,42 @@
 [![Build Status](https://travis-ci.org/tikv/rust-prometheus.svg?branch=master)](https://travis-ci.org/pingcap/rust-prometheus)
 [![docs.rs](https://docs.rs/prometheus/badge.svg)](https://docs.rs/prometheus)
 [![crates.io](http://meritbadge.herokuapp.com/prometheus)](https://crates.io/crates/prometheus)
-[![Dependency Status](https://deps.rs/repo/github/tikv/rust-prometheus/status.svg)](https://deps.rs/repo/github/tikv/rust-prometheus)
 
 This is the [Rust](https://www.rust-lang.org) client library for
-[Prometheus](http://prometheus.io). The main Structures and APIs are ported from
-[Go client](https://github.com/prometheus/client_golang).
+[Prometheus](http://prometheus.io). The main data structures and APIs are ported
+from [Go client](https://github.com/prometheus/client_golang).
 
 
-### Documentation
+## Documentation
 
 Find the latest documentation at https://docs.rs/prometheus
 
 
+## Advanced
+
+### Features
+
+This library supports four features:
+
+- `gen`: To generate protobuf client with the latest protobuf version instead of
+  using the pre-generated client.
+
+- `nightly`: Enable nightly only features.
+
+- `push`: Enable push support.
+
+- `process`: For collecting process info.
+
+- `push`: Enable push support.
+
+
+### Static Metric
+
+Static metric helps you make metric vectors faster.
```

Comment:
You are selling this one very well, but it doesn't tell much. What about adding "by allowing caching frequently label sets" (or whatever the proper term for that is)?

Reply:
Just moved this from the existing [README.md](https://github.com/tikv/rust-prometheus/blob/master/README.md#advanced). :+1: Extended with more details.